src/tile.js

```diff
@@ -268,6 +268,7 @@ $.Tile.prototype = /** @lends OpenSeadragon.Tile.prototype */{
         context.save();
 
         context.globalAlpha = this.opacity;
+        context.globalCompositeOperation = this.compositeOperation;
```

Comment:
I don't think we want this line, right? Tile objects don't have this new property, unless I'm missing something.